docs/version_description.rst

```diff
@@ -0,0 +1,35 @@
+###################
+Version Description
+###################
+
+v1.0 - v1.4
+------------
+
+The original IDL version of PAHFIT can be found in: `http://tir.astro.utoledo.edu/jdsmith/research/pahfit.php <http://tir.astro.utoledo.edu/jdsmith/research/pahfit.php>`_. For more details and background of ``PAHFIT``, see the `background <https://pahfit.readthedocs.io/en/latest/background.html>`_ page.
+
+v2.0
+------------
+
+A direct translation of python PAHFIT from the IDL version based on the astropy.modeling framework. Also, the modular approach has been built-in for future contributions, e.g., science/instrument packs. 
```

Comment:
I don't think we should say it is a direct translation.  We have attempted to match the capabilities of the IDL version, but we didn't do this by directly translating the IDL code to python.  This is implied by the 2nd sentence as we have actually made significant changes to make the python version more flexible than the IDL version already.